GameData/ROCapsules/Assets/BDB/Gemini/Paint/TTV/TTV.cfg

```diff
@@ -0,0 +1,67 @@
+@PART[ROC-GeminiCM]:FOR[Bluedog_DB]
```

Comment:
Do not include :FOR[SomeOtherMod] passes.  Must be :BEFORE or :AFTER.  Issue repeats throughout this file.

Reply:
Also, is the part `ROC-GeminiCMBDB` or `ROC-GeminiCM`?